2024-03-03 23:59:40.008 [info] (Client) Pylance async client (2024.2.2) started with python extension (2024.2.1)
2024-03-03 23:59:40.418 [info] [Info  - 11:59:40 PM] (22848) Pylance language server 2024.2.2 (pyright version 1.1.348, commit cfb1de0c) starting
2024-03-03 23:59:40.418 [info] [Info  - 11:59:40 PM] (22848) Server root directory: file:///home/andrew/.vscode/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-03-04 00:00:07.600 [info] [Info  - 12:00:07 AM] (22848) Starting service instance "<default>"
2024-03-04 00:00:07.684 [info] [Info  - 12:00:07 AM] (22848) Setting pythonPath for service "<default>": "/bin/python"
2024-03-04 00:00:07.866 [info] [Info  - 12:00:07 AM] (22848) Setting environmentName for service "<default>": "3.11.7 (global)"
2024-03-04 00:00:07.867 [info] [Info  - 12:00:07 AM] (22848) Assuming Python version 3.11
2024-03-04 00:00:07.868 [info] [Info  - 12:00:07 AM] (22848) No source files found.
2024-03-04 00:00:08.037 [info] [Info  - 12:00:08 AM] (22848) Background analysis(1) root directory: file:///home/andrew/.vscode/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-03-04 00:00:08.039 [info] [Info  - 12:00:08 AM] (22848) Background analysis(1) started
